commerce-product-type-virtual-service/src/main/java/com/liferay/commerce/product/type/virtual/internal/upgrade/v1_1_0/CPDefinitionVirtualSettingUpgradeProcess.java

```diff
@@ -48,9 +48,7 @@ protected void doUpgrade() throws Exception {
 			String template = StringUtil.read(
 				CPDefinitionVirtualSettingUpgradeProcess.class.
 					getResourceAsStream(
-						"/META-INF/com/liferay/commerce/product/type/virtual" +
-							"/internal/upgrade/v1_1_0/dependencies" +
-								"/CPDefinitionVirtualSetting.sql"));
+						"/dependencies/CPDefinitionVirtualSetting.sql"));
```

Comment:
Hey @Commando18, are you sure this String should start with a `/`? I think if you're doing a relative path to the resource it should just be `"dependencies/CPDefinitionVirtualSetting.sql"`...

Reply:
Hey @ethanbustad, you're right. I'll change it, thanks!